[2022-08-02 02:41:38,141] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: Relasi_karyawan.load_data 2022-08-02T02:41:29.820934+00:00 [queued]>
[2022-08-02 02:41:38,163] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: Relasi_karyawan.load_data 2022-08-02T02:41:29.820934+00:00 [queued]>
[2022-08-02 02:41:38,163] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 02:41:38,163] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-08-02 02:41:38,164] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 02:41:38,206] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): load_data> on 2022-08-02T02:41:29.820934+00:00
[2022-08-02 02:41:38,211] {standard_task_runner.py:52} INFO - Started process 84 to run task
[2022-08-02 02:41:38,218] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Relasi_karyawan', 'load_data', '2022-08-02T02:41:29.820934+00:00', '--job-id', '14', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmprsark1ty', '--error-file', '/tmp/tmpb3w23mxq']
[2022-08-02 02:41:38,219] {standard_task_runner.py:77} INFO - Job 14: Subtask load_data
[2022-08-02 02:41:38,296] {logging_mixin.py:104} INFO - Running <TaskInstance: Relasi_karyawan.load_data 2022-08-02T02:41:29.820934+00:00 [running]> on host 02d6282e7057
[2022-08-02 02:41:38,386] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Relasi_karyawan
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T02:41:29.820934+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-02T02:41:29.820934+00:00
[2022-08-02 02:41:38,600] {python.py:151} INFO - Done. Returned value was: None
[2022-08-02 02:41:38,612] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=Relasi_karyawan, task_id=load_data, execution_date=20220802T024129, start_date=20220802T024138, end_date=20220802T024138
[2022-08-02 02:41:38,664] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-02 02:41:38,710] {local_task_job.py:151} INFO - Task exited with return code 0
